Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -2674,6 +2674,21 @@ EventHandlerDrivenSmoothKeyboardScrollingEnabled:
     WebCore:
       default: false
 
+EventTimingEnabled:
+  type: bool
+  status: internal
```

testable is probably what you want, because you want start getting test rebaselines

---

Enabling the flag currently is causing some tests in `web-platform-tests/event-timing/` to timeout instead of failing early, making the entire suite take over 20mins (from 25s). Because of that, ~I'm leaning towards keeping this `internal` (or maybe `unstable`) until that is sorted out~ will follow @rreno 's suggestion to skip and rebaseline the offending tests

---

You can just skip the tests for now in LayoutTests/TestExpectations